content/developers/developer-patterns/3rdparty-verification/index.md

```diff
@@ -19,140 +19,123 @@ aliases:
 ## Overview
 
 The DataTrails ledger is a log that can be distributed.
-
-When a replicated copy of your merkle log is held by an independent party,
-it is impossible for DataTrails to modify your log to refute your claims.
+When a replicated copy of a merkle log is held by an independent party, it is impossible for DataTrails to modify a log to refute claims.
 
 A full replica is not necessary in many cases.
 DataTrails provides a merkle log for each tenant.
 Only the portions of the logs that attest to records that are still of interest need be retained.
 And only for the specific tenants of interest.
 
-This article provides the information needed to address three scenarios.
+This article provides the information needed to address three scenarios:
 
-1. You want to detect if DataTrails improperly updates the log, and prove that to others.
-2. You want to be able to produce proof that an event is in your log after DataTrails has improperly updated their copy.
-3. You want fully independent verification of your events, without recourse to DataTrails.
+1. Detect if DataTrails improperly updates the log, and prove that to others
+1. Be able to produce proof that an event is in a log after DataTrails has improperly updated their copy
+1. Full independent verification of events, without recourse to DataTrails
 
-Each of these can be accomplished using the veracity `replicate-logs` and
-`watch` commands to check the log operation and replicate some or all of your
-log data.
+Each of these can be accomplished using the veracity `replicate-logs` and `watch` commands to check the log operation and replicate some or all of log data.
 
 The first two goals require replication and verification of at least the most recently updated log section.
-
 The second goal requires retention of the local log sections verifying any event that is still of value.
+The last goal, full independent *verification*, requires retention of the verifiable log data and the ability to reproduce the original event.
 
-The last goal, full independent *verification*,
-requires retention of the verifiable log data and the ability to reproduce the original event.
+With a trusted local copy of the verifiable data, even after you detect a tamper, you can chose to rely on DataTrails storage of your event.
 
-Because you have a trusted local copy of the verifiable data, even after you
-detect a tamper, you can chose to rely on DataTrails storage of your event.
+When the event is fetched, if it can be verified against local log data, knowing that the DataTrails log remains correct.
```

Comment:
```suggestion
When the event is fetched, if it can be verified against local log data, you have proven that the DataTrails storage of your event remains correct.
```